WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1541,8 +1544,10 @@ private void showGutenbergInformativeDialog() {
     private void setGutenbergEnabledIfNeeded() {
         if (AppPrefs.isGutenbergAutoEnabledForTheNewPosts()
             && !mIsNewPost
-            && !AppPrefs.isGutenbergDefaultForNewPosts()) {
+            && !SiteUtils.isBlockEditorDefaultForNewPost(mSite, mAccountStore)) {
             AppPrefs.setGutenbergDefaultForNewPosts(true);
```

Comment:
Do we actually need the app-wide Gutenberg pref anymore? Can we just remove it altogether?

Reply:
No, we don't need it anymore. Removed.

Reply:
Removed with 5680f4da63f96907a02755aacf6b4893d190a998.